protocol/messenger_sync_saved_addresses_test.go

```diff
@@ -0,0 +1,248 @@
+package protocol
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/status-im/status-go/protocol/encryption/multidevice"
+	"github.com/status-im/status-go/protocol/tt"
+	"github.com/status-im/status-go/services/wallet"
+	"github.com/status-im/status-go/waku"
+
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
+
+	"github.com/status-im/status-go/eth-node/types"
```

Comment:
Group imports like this:

- standard lib
- 3rd party libs
- go-ethereum & status-go libs

```suggestion
	"github.com/stretchr/testify/suite"
	"go.uber.org/zap"
	
	"github.com/ethereum/go-ethereum/common"
	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
	"github.com/status-im/status-go/eth-node/crypto"
	"github.com/status-im/status-go/eth-node/types"
	"github.com/status-im/status-go/protocol/encryption/multidevice"
	"github.com/status-im/status-go/protocol/tt"
	"github.com/status-im/status-go/services/wallet"
	"github.com/status-im/status-go/waku"
```